/short/kl02/vhl548/miniconda3/envs/radar/lib/python3.6/site-packages/pyart/config.py:22: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
  import imp
/short/kl02/vhl548/miniconda3/envs/radar/lib/python3.6/site-packages/h5py/__init__.py:36: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
  from ._conv import register_converters as _register_converters
Traceback (most recent call last):
  File "daily_file.py", line 245, in multiproc_buffer
    processing_line(input_dir, output_dir, year, month, day)
  File "daily_file.py", line 128, in processing_line
    mymoment = get_moment(radar, mykey, fillvalue=FILLVALUE)
  File "/home/548/vhl548/projects/RADAR_codes/cpol_level_2/scripts/processing_codes/utils.py", line 166, in get_moment
    mymoment = radar.fields[moment_name]
KeyError: 'cross_correlation_ratio'
Traceback (most recent call last):
  File "daily_file.py", line 245, in multiproc_buffer
    processing_line(input_dir, output_dir, year, month, day)
  File "daily_file.py", line 128, in processing_line
    mymoment = get_moment(radar, mykey, fillvalue=FILLVALUE)
  File "/home/548/vhl548/projects/RADAR_codes/cpol_level_2/scripts/processing_codes/utils.py", line 166, in get_moment
    mymoment = radar.fields[moment_name]
KeyError: 'cross_correlation_ratio'
Traceback (most recent call last):
  File "daily_file.py", line 245, in multiproc_buffer
    processing_line(input_dir, output_dir, year, month, day)
  File "daily_file.py", line 128, in processing_line
    mymoment = get_moment(radar, mykey, fillvalue=FILLVALUE)
  File "/home/548/vhl548/projects/RADAR_codes/cpol_level_2/scripts/processing_codes/utils.py", line 166, in get_moment
    mymoment = radar.fields[moment_name]
KeyError: 'cross_correlation_ratio'
Traceback (most recent call last):
  File "daily_file.py", line 245, in multiproc_buffer
    processing_line(input_dir, output_dir, year, month, day)
  File "daily_file.py", line 128, in processing_line
    mymoment = get_moment(radar, mykey, fillvalue=FILLVALUE)
  File "/home/548/vhl548/projects/RADAR_codes/cpol_level_2/scripts/processing_codes/utils.py", line 166, in get_moment
    mymoment = radar.fields[moment_name]
KeyError: 'cross_correlation_ratio'
Traceback (most recent call last):
  File "daily_file.py", line 245, in multiproc_buffer
    processing_line(input_dir, output_dir, year, month, day)
  File "daily_file.py", line 128, in processing_line
    mymoment = get_moment(radar, mykey, fillvalue=FILLVALUE)
  File "/home/548/vhl548/projects/RADAR_codes/cpol_level_2/scripts/processing_codes/utils.py", line 166, in get_moment
    mymoment = radar.fields[moment_name]
KeyError: 'cross_correlation_ratio'
Traceback (most recent call last):
  File "daily_file.py", line 245, in multiproc_buffer
    processing_line(input_dir, output_dir, year, month, day)
  File "daily_file.py", line 128, in processing_line
    mymoment = get_moment(radar, mykey, fillvalue=FILLVALUE)
  File "/home/548/vhl548/projects/RADAR_codes/cpol_level_2/scripts/processing_codes/utils.py", line 166, in get_moment
    mymoment = radar.fields[moment_name]
KeyError: 'cross_correlation_ratio'
Traceback (most recent call last):
  File "daily_file.py", line 245, in multiproc_buffer
    processing_line(input_dir, output_dir, year, month, day)
  File "daily_file.py", line 128, in processing_line
    mymoment = get_moment(radar, mykey, fillvalue=FILLVALUE)
  File "/home/548/vhl548/projects/RADAR_codes/cpol_level_2/scripts/processing_codes/utils.py", line 166, in get_moment
    mymoment = radar.fields[moment_name]
KeyError: 'cross_correlation_ratio'
Traceback (most recent call last):
  File "daily_file.py", line 245, in multiproc_buffer
    processing_line(input_dir, output_dir, year, month, day)
  File "daily_file.py", line 128, in processing_line
    mymoment = get_moment(radar, mykey, fillvalue=FILLVALUE)
  File "/home/548/vhl548/projects/RADAR_codes/cpol_level_2/scripts/processing_codes/utils.py", line 166, in get_moment
    mymoment = radar.fields[moment_name]
KeyError: 'cross_correlation_ratio'
Traceback (most recent call last):
  File "daily_file.py", line 245, in multiproc_buffer
    processing_line(input_dir, output_dir, year, month, day)
  File "daily_file.py", line 128, in processing_line
    mymoment = get_moment(radar, mykey, fillvalue=FILLVALUE)
  File "/home/548/vhl548/projects/RADAR_codes/cpol_level_2/scripts/processing_codes/utils.py", line 166, in get_moment
    mymoment = radar.fields[moment_name]
KeyError: 'cross_correlation_ratio'
Traceback (most recent call last):
  File "daily_file.py", line 245, in multiproc_buffer
    processing_line(input_dir, output_dir, year, month, day)
  File "daily_file.py", line 128, in processing_line
    mymoment = get_moment(radar, mykey, fillvalue=FILLVALUE)
  File "/home/548/vhl548/projects/RADAR_codes/cpol_level_2/scripts/processing_codes/utils.py", line 166, in get_moment
    mymoment = radar.fields[moment_name]
KeyError: 'cross_correlation_ratio'
Traceback (most recent call last):
  File "daily_file.py", line 245, in multiproc_buffer
    processing_line(input_dir, output_dir, year, month, day)
  File "daily_file.py", line 128, in processing_line
    mymoment = get_moment(radar, mykey, fillvalue=FILLVALUE)
  File "/home/548/vhl548/projects/RADAR_codes/cpol_level_2/scripts/processing_codes/utils.py", line 166, in get_moment
    mymoment = radar.fields[moment_name]
KeyError: 'cross_correlation_ratio'
Traceback (most recent call last):
  File "daily_file.py", line 245, in multiproc_buffer
    processing_line(input_dir, output_dir, year, month, day)
  File "daily_file.py", line 128, in processing_line
    mymoment = get_moment(radar, mykey, fillvalue=FILLVALUE)
  File "/home/548/vhl548/projects/RADAR_codes/cpol_level_2/scripts/processing_codes/utils.py", line 166, in get_moment
    mymoment = radar.fields[moment_name]
KeyError: 'cross_correlation_ratio'
Traceback (most recent call last):
  File "daily_file.py", line 245, in multiproc_buffer
    processing_line(input_dir, output_dir, year, month, day)
  File "daily_file.py", line 128, in processing_line
    mymoment = get_moment(radar, mykey, fillvalue=FILLVALUE)
  File "/home/548/vhl548/projects/RADAR_codes/cpol_level_2/scripts/processing_codes/utils.py", line 166, in get_moment
    mymoment = radar.fields[moment_name]
KeyError: 'cross_correlation_ratio'
Traceback (most recent call last):
  File "daily_file.py", line 245, in multiproc_buffer
    processing_line(input_dir, output_dir, year, month, day)
  File "daily_file.py", line 128, in processing_line
    mymoment = get_moment(radar, mykey, fillvalue=FILLVALUE)
  File "/home/548/vhl548/projects/RADAR_codes/cpol_level_2/scripts/processing_codes/utils.py", line 166, in get_moment
    mymoment = radar.fields[moment_name]
KeyError: 'cross_correlation_ratio'
Traceback (most recent call last):
  File "daily_file.py", line 245, in multiproc_buffer
    processing_line(input_dir, output_dir, year, month, day)
  File "daily_file.py", line 128, in processing_line
    mymoment = get_moment(radar, mykey, fillvalue=FILLVALUE)
  File "/home/548/vhl548/projects/RADAR_codes/cpol_level_2/scripts/processing_codes/utils.py", line 166, in get_moment
    mymoment = radar.fields[moment_name]
KeyError: 'cross_correlation_ratio'
Traceback (most recent call last):
  File "daily_file.py", line 245, in multiproc_buffer
    processing_line(input_dir, output_dir, year, month, day)
  File "daily_file.py", line 128, in processing_line
    mymoment = get_moment(radar, mykey, fillvalue=FILLVALUE)
  File "/home/548/vhl548/projects/RADAR_codes/cpol_level_2/scripts/processing_codes/utils.py", line 166, in get_moment
    mymoment = radar.fields[moment_name]
KeyError: 'cross_correlation_ratio'
Traceback (most recent call last):
  File "daily_file.py", line 245, in multiproc_buffer
    processing_line(input_dir, output_dir, year, month, day)
  File "daily_file.py", line 128, in processing_line
    mymoment = get_moment(radar, mykey, fillvalue=FILLVALUE)
  File "/home/548/vhl548/projects/RADAR_codes/cpol_level_2/scripts/processing_codes/utils.py", line 166, in get_moment
    mymoment = radar.fields[moment_name]
KeyError: 'cross_correlation_ratio'
Traceback (most recent call last):
  File "daily_file.py", line 245, in multiproc_buffer
    processing_line(input_dir, output_dir, year, month, day)
  File "daily_file.py", line 128, in processing_line
    mymoment = get_moment(radar, mykey, fillvalue=FILLVALUE)
  File "/home/548/vhl548/projects/RADAR_codes/cpol_level_2/scripts/processing_codes/utils.py", line 166, in get_moment
    mymoment = radar.fields[moment_name]
KeyError: 'cross_correlation_ratio'
Traceback (most recent call last):
  File "daily_file.py", line 245, in multiproc_buffer
    processing_line(input_dir, output_dir, year, month, day)
  File "daily_file.py", line 128, in processing_line
    mymoment = get_moment(radar, mykey, fillvalue=FILLVALUE)
  File "/home/548/vhl548/projects/RADAR_codes/cpol_level_2/scripts/processing_codes/utils.py", line 166, in get_moment
    mymoment = radar.fields[moment_name]
KeyError: 'cross_correlation_ratio'
Traceback (most recent call last):
  File "daily_file.py", line 245, in multiproc_buffer
    processing_line(input_dir, output_dir, year, month, day)
  File "daily_file.py", line 128, in processing_line
    mymoment = get_moment(radar, mykey, fillvalue=FILLVALUE)
  File "/home/548/vhl548/projects/RADAR_codes/cpol_level_2/scripts/processing_codes/utils.py", line 166, in get_moment
    mymoment = radar.fields[moment_name]
KeyError: 'cross_correlation_ratio'
Traceback (most recent call last):
  File "daily_file.py", line 245, in multiproc_buffer
    processing_line(input_dir, output_dir, year, month, day)
  File "daily_file.py", line 128, in processing_line
    mymoment = get_moment(radar, mykey, fillvalue=FILLVALUE)
  File "/home/548/vhl548/projects/RADAR_codes/cpol_level_2/scripts/processing_codes/utils.py", line 166, in get_moment
    mymoment = radar.fields[moment_name]
KeyError: 'cross_correlation_ratio'
Traceback (most recent call last):
  File "daily_file.py", line 245, in multiproc_buffer
    processing_line(input_dir, output_dir, year, month, day)
  File "daily_file.py", line 128, in processing_line
    mymoment = get_moment(radar, mykey, fillvalue=FILLVALUE)
  File "/home/548/vhl548/projects/RADAR_codes/cpol_level_2/scripts/processing_codes/utils.py", line 166, in get_moment
    mymoment = radar.fields[moment_name]
KeyError: 'cross_correlation_ratio'
Traceback (most recent call last):
  File "daily_file.py", line 245, in multiproc_buffer
    processing_line(input_dir, output_dir, year, month, day)
  File "daily_file.py", line 128, in processing_line
    mymoment = get_moment(radar, mykey, fillvalue=FILLVALUE)
  File "/home/548/vhl548/projects/RADAR_codes/cpol_level_2/scripts/processing_codes/utils.py", line 166, in get_moment
    mymoment = radar.fields[moment_name]
KeyError: 'cross_correlation_ratio'
Traceback (most recent call last):
  File "daily_file.py", line 245, in multiproc_buffer
    processing_line(input_dir, output_dir, year, month, day)
  File "daily_file.py", line 128, in processing_line
    mymoment = get_moment(radar, mykey, fillvalue=FILLVALUE)
  File "/home/548/vhl548/projects/RADAR_codes/cpol_level_2/scripts/processing_codes/utils.py", line 166, in get_moment
    mymoment = radar.fields[moment_name]
KeyError: 'cross_correlation_ratio'
Traceback (most recent call last):
  File "daily_file.py", line 245, in multiproc_buffer
    processing_line(input_dir, output_dir, year, month, day)
  File "daily_file.py", line 128, in processing_line
    mymoment = get_moment(radar, mykey, fillvalue=FILLVALUE)
  File "/home/548/vhl548/projects/RADAR_codes/cpol_level_2/scripts/processing_codes/utils.py", line 166, in get_moment
    mymoment = radar.fields[moment_name]
KeyError: 'cross_correlation_ratio'
Traceback (most recent call last):
  File "daily_file.py", line 245, in multiproc_buffer
    processing_line(input_dir, output_dir, year, month, day)
  File "daily_file.py", line 128, in processing_line
    mymoment = get_moment(radar, mykey, fillvalue=FILLVALUE)
  File "/home/548/vhl548/projects/RADAR_codes/cpol_level_2/scripts/processing_codes/utils.py", line 166, in get_moment
    mymoment = radar.fields[moment_name]
KeyError: 'cross_correlation_ratio'
Traceback (most recent call last):
  File "daily_file.py", line 245, in multiproc_buffer
    processing_line(input_dir, output_dir, year, month, day)
  File "daily_file.py", line 128, in processing_line
    mymoment = get_moment(radar, mykey, fillvalue=FILLVALUE)
  File "/home/548/vhl548/projects/RADAR_codes/cpol_level_2/scripts/processing_codes/utils.py", line 166, in get_moment
    mymoment = radar.fields[moment_name]
KeyError: 'cross_correlation_ratio'
Traceback (most recent call last):
  File "daily_file.py", line 245, in multiproc_buffer
    processing_line(input_dir, output_dir, year, month, day)
  File "daily_file.py", line 128, in processing_line
    mymoment = get_moment(radar, mykey, fillvalue=FILLVALUE)
  File "/home/548/vhl548/projects/RADAR_codes/cpol_level_2/scripts/processing_codes/utils.py", line 166, in get_moment
    mymoment = radar.fields[moment_name]
KeyError: 'cross_correlation_ratio'
Traceback (most recent call last):
  File "daily_file.py", line 245, in multiproc_buffer
    processing_line(input_dir, output_dir, year, month, day)
  File "daily_file.py", line 128, in processing_line
    mymoment = get_moment(radar, mykey, fillvalue=FILLVALUE)
  File "/home/548/vhl548/projects/RADAR_codes/cpol_level_2/scripts/processing_codes/utils.py", line 166, in get_moment
    mymoment = radar.fields[moment_name]
KeyError: 'cross_correlation_ratio'
Traceback (most recent call last):
  File "daily_file.py", line 245, in multiproc_buffer
    processing_line(input_dir, output_dir, year, month, day)
  File "daily_file.py", line 128, in processing_line
    mymoment = get_moment(radar, mykey, fillvalue=FILLVALUE)
  File "/home/548/vhl548/projects/RADAR_codes/cpol_level_2/scripts/processing_codes/utils.py", line 166, in get_moment
    mymoment = radar.fields[moment_name]
KeyError: 'cross_correlation_ratio'
Traceback (most recent call last):
  File "daily_file.py", line 245, in multiproc_buffer
    processing_line(input_dir, output_dir, year, month, day)
  File "daily_file.py", line 128, in processing_line
    mymoment = get_moment(radar, mykey, fillvalue=FILLVALUE)
  File "/home/548/vhl548/projects/RADAR_codes/cpol_level_2/scripts/processing_codes/utils.py", line 166, in get_moment
    mymoment = radar.fields[moment_name]
KeyError: 'cross_correlation_ratio'
Traceback (most recent call last):
  File "daily_file.py", line 245, in multiproc_buffer
    processing_line(input_dir, output_dir, year, month, day)
  File "daily_file.py", line 128, in processing_line
    mymoment = get_moment(radar, mykey, fillvalue=FILLVALUE)
  File "/home/548/vhl548/projects/RADAR_codes/cpol_level_2/scripts/processing_codes/utils.py", line 166, in get_moment
    mymoment = radar.fields[moment_name]
KeyError: 'cross_correlation_ratio'
Traceback (most recent call last):
  File "daily_file.py", line 245, in multiproc_buffer
    processing_line(input_dir, output_dir, year, month, day)
  File "daily_file.py", line 128, in processing_line
    mymoment = get_moment(radar, mykey, fillvalue=FILLVALUE)
  File "/home/548/vhl548/projects/RADAR_codes/cpol_level_2/scripts/processing_codes/utils.py", line 166, in get_moment
    mymoment = radar.fields[moment_name]
KeyError: 'cross_correlation_ratio'
Traceback (most recent call last):
  File "daily_file.py", line 245, in multiproc_buffer
    processing_line(input_dir, output_dir, year, month, day)
  File "daily_file.py", line 128, in processing_line
    mymoment = get_moment(radar, mykey, fillvalue=FILLVALUE)
  File "/home/548/vhl548/projects/RADAR_codes/cpol_level_2/scripts/processing_codes/utils.py", line 166, in get_moment
    mymoment = radar.fields[moment_name]
KeyError: 'cross_correlation_ratio'
Traceback (most recent call last):
  File "daily_file.py", line 245, in multiproc_buffer
    processing_line(input_dir, output_dir, year, month, day)
  File "daily_file.py", line 128, in processing_line
    mymoment = get_moment(radar, mykey, fillvalue=FILLVALUE)
  File "/home/548/vhl548/projects/RADAR_codes/cpol_level_2/scripts/processing_codes/utils.py", line 166, in get_moment
    mymoment = radar.fields[moment_name]
KeyError: 'cross_correlation_ratio'
Traceback (most recent call last):
  File "daily_file.py", line 245, in multiproc_buffer
    processing_line(input_dir, output_dir, year, month, day)
  File "daily_file.py", line 128, in processing_line
    mymoment = get_moment(radar, mykey, fillvalue=FILLVALUE)
  File "/home/548/vhl548/projects/RADAR_codes/cpol_level_2/scripts/processing_codes/utils.py", line 166, in get_moment
    mymoment = radar.fields[moment_name]
KeyError: 'cross_correlation_ratio'
Traceback (most recent call last):
  File "daily_file.py", line 245, in multiproc_buffer
    processing_line(input_dir, output_dir, year, month, day)
  File "daily_file.py", line 128, in processing_line
    mymoment = get_moment(radar, mykey, fillvalue=FILLVALUE)
  File "/home/548/vhl548/projects/RADAR_codes/cpol_level_2/scripts/processing_codes/utils.py", line 166, in get_moment
    mymoment = radar.fields[moment_name]
KeyError: 'cross_correlation_ratio'
Traceback (most recent call last):
  File "daily_file.py", line 245, in multiproc_buffer
    processing_line(input_dir, output_dir, year, month, day)
  File "daily_file.py", line 128, in processing_line
    mymoment = get_moment(radar, mykey, fillvalue=FILLVALUE)
  File "/home/548/vhl548/projects/RADAR_codes/cpol_level_2/scripts/processing_codes/utils.py", line 166, in get_moment
    mymoment = radar.fields[moment_name]
KeyError: 'cross_correlation_ratio'
Traceback (most recent call last):
  File "daily_file.py", line 245, in multiproc_buffer
    processing_line(input_dir, output_dir, year, month, day)
  File "daily_file.py", line 128, in processing_line
    mymoment = get_moment(radar, mykey, fillvalue=FILLVALUE)
  File "/home/548/vhl548/projects/RADAR_codes/cpol_level_2/scripts/processing_codes/utils.py", line 166, in get_moment
    mymoment = radar.fields[moment_name]
KeyError: 'cross_correlation_ratio'
Traceback (most recent call last):
  File "daily_file.py", line 245, in multiproc_buffer
    processing_line(input_dir, output_dir, year, month, day)
  File "daily_file.py", line 128, in processing_line
    mymoment = get_moment(radar, mykey, fillvalue=FILLVALUE)
  File "/home/548/vhl548/projects/RADAR_codes/cpol_level_2/scripts/processing_codes/utils.py", line 166, in get_moment
    mymoment = radar.fields[moment_name]
KeyError: 'cross_correlation_ratio'
Traceback (most recent call last):
  File "daily_file.py", line 245, in multiproc_buffer
    processing_line(input_dir, output_dir, year, month, day)
  File "daily_file.py", line 128, in processing_line
    mymoment = get_moment(radar, mykey, fillvalue=FILLVALUE)
  File "/home/548/vhl548/projects/RADAR_codes/cpol_level_2/scripts/processing_codes/utils.py", line 166, in get_moment
    mymoment = radar.fields[moment_name]
KeyError: 'cross_correlation_ratio'
Traceback (most recent call last):
  File "daily_file.py", line 245, in multiproc_buffer
    processing_line(input_dir, output_dir, year, month, day)
  File "daily_file.py", line 128, in processing_line
    mymoment = get_moment(radar, mykey, fillvalue=FILLVALUE)
  File "/home/548/vhl548/projects/RADAR_codes/cpol_level_2/scripts/processing_codes/utils.py", line 166, in get_moment
    mymoment = radar.fields[moment_name]
KeyError: 'cross_correlation_ratio'
Traceback (most recent call last):
  File "daily_file.py", line 245, in multiproc_buffer
    processing_line(input_dir, output_dir, year, month, day)
  File "daily_file.py", line 128, in processing_line
    mymoment = get_moment(radar, mykey, fillvalue=FILLVALUE)
  File "/home/548/vhl548/projects/RADAR_codes/cpol_level_2/scripts/processing_codes/utils.py", line 166, in get_moment
    mymoment = radar.fields[moment_name]
KeyError: 'cross_correlation_ratio'
Traceback (most recent call last):
  File "daily_file.py", line 245, in multiproc_buffer
    processing_line(input_dir, output_dir, year, month, day)
  File "daily_file.py", line 128, in processing_line
    mymoment = get_moment(radar, mykey, fillvalue=FILLVALUE)
  File "/home/548/vhl548/projects/RADAR_codes/cpol_level_2/scripts/processing_codes/utils.py", line 166, in get_moment
    mymoment = radar.fields[moment_name]
KeyError: 'cross_correlation_ratio'
Traceback (most recent call last):
  File "daily_file.py", line 245, in multiproc_buffer
    processing_line(input_dir, output_dir, year, month, day)
  File "daily_file.py", line 128, in processing_line
    mymoment = get_moment(radar, mykey, fillvalue=FILLVALUE)
  File "/home/548/vhl548/projects/RADAR_codes/cpol_level_2/scripts/processing_codes/utils.py", line 166, in get_moment
    mymoment = radar.fields[moment_name]
KeyError: 'cross_correlation_ratio'
Traceback (most recent call last):
  File "daily_file.py", line 245, in multiproc_buffer
    processing_line(input_dir, output_dir, year, month, day)
  File "daily_file.py", line 128, in processing_line
    mymoment = get_moment(radar, mykey, fillvalue=FILLVALUE)
  File "/home/548/vhl548/projects/RADAR_codes/cpol_level_2/scripts/processing_codes/utils.py", line 166, in get_moment
    mymoment = radar.fields[moment_name]
KeyError: 'cross_correlation_ratio'
Traceback (most recent call last):
  File "daily_file.py", line 245, in multiproc_buffer
    processing_line(input_dir, output_dir, year, month, day)
  File "daily_file.py", line 128, in processing_line
    mymoment = get_moment(radar, mykey, fillvalue=FILLVALUE)
  File "/home/548/vhl548/projects/RADAR_codes/cpol_level_2/scripts/processing_codes/utils.py", line 166, in get_moment
    mymoment = radar.fields[moment_name]
KeyError: 'cross_correlation_ratio'
Traceback (most recent call last):
  File "daily_file.py", line 245, in multiproc_buffer
    processing_line(input_dir, output_dir, year, month, day)
  File "daily_file.py", line 128, in processing_line
    mymoment = get_moment(radar, mykey, fillvalue=FILLVALUE)
  File "/home/548/vhl548/projects/RADAR_codes/cpol_level_2/scripts/processing_codes/utils.py", line 166, in get_moment
    mymoment = radar.fields[moment_name]
KeyError: 'cross_correlation_ratio'
Traceback (most recent call last):
  File "daily_file.py", line 245, in multiproc_buffer
    processing_line(input_dir, output_dir, year, month, day)
  File "daily_file.py", line 128, in processing_line
    mymoment = get_moment(radar, mykey, fillvalue=FILLVALUE)
  File "/home/548/vhl548/projects/RADAR_codes/cpol_level_2/scripts/processing_codes/utils.py", line 166, in get_moment
    mymoment = radar.fields[moment_name]
KeyError: 'cross_correlation_ratio'
Traceback (most recent call last):
  File "daily_file.py", line 245, in multiproc_buffer
    processing_line(input_dir, output_dir, year, month, day)
  File "daily_file.py", line 128, in processing_line
    mymoment = get_moment(radar, mykey, fillvalue=FILLVALUE)
  File "/home/548/vhl548/projects/RADAR_codes/cpol_level_2/scripts/processing_codes/utils.py", line 166, in get_moment
    mymoment = radar.fields[moment_name]
KeyError: 'cross_correlation_ratio'
Traceback (most recent call last):
  File "daily_file.py", line 245, in multiproc_buffer
    processing_line(input_dir, output_dir, year, month, day)
  File "daily_file.py", line 128, in processing_line
    mymoment = get_moment(radar, mykey, fillvalue=FILLVALUE)
  File "/home/548/vhl548/projects/RADAR_codes/cpol_level_2/scripts/processing_codes/utils.py", line 166, in get_moment
    mymoment = radar.fields[moment_name]
KeyError: 'cross_correlation_ratio'
Traceback (most recent call last):
  File "daily_file.py", line 245, in multiproc_buffer
    processing_line(input_dir, output_dir, year, month, day)
  File "daily_file.py", line 128, in processing_line
    mymoment = get_moment(radar, mykey, fillvalue=FILLVALUE)
  File "/home/548/vhl548/projects/RADAR_codes/cpol_level_2/scripts/processing_codes/utils.py", line 166, in get_moment
    mymoment = radar.fields[moment_name]
KeyError: 'cross_correlation_ratio'
Traceback (most recent call last):
  File "daily_file.py", line 245, in multiproc_buffer
    processing_line(input_dir, output_dir, year, month, day)
  File "daily_file.py", line 128, in processing_line
    mymoment = get_moment(radar, mykey, fillvalue=FILLVALUE)
  File "/home/548/vhl548/projects/RADAR_codes/cpol_level_2/scripts/processing_codes/utils.py", line 166, in get_moment
    mymoment = radar.fields[moment_name]
KeyError: 'cross_correlation_ratio'
Traceback (most recent call last):
  File "daily_file.py", line 245, in multiproc_buffer
    processing_line(input_dir, output_dir, year, month, day)
  File "daily_file.py", line 128, in processing_line
    mymoment = get_moment(radar, mykey, fillvalue=FILLVALUE)
  File "/home/548/vhl548/projects/RADAR_codes/cpol_level_2/scripts/processing_codes/utils.py", line 166, in get_moment
    mymoment = radar.fields[moment_name]
KeyError: 'cross_correlation_ratio'
Traceback (most recent call last):
  File "daily_file.py", line 245, in multiproc_buffer
    processing_line(input_dir, output_dir, year, month, day)
  File "daily_file.py", line 128, in processing_line
    mymoment = get_moment(radar, mykey, fillvalue=FILLVALUE)
  File "/home/548/vhl548/projects/RADAR_codes/cpol_level_2/scripts/processing_codes/utils.py", line 166, in get_moment
    mymoment = radar.fields[moment_name]
KeyError: 'cross_correlation_ratio'
Traceback (most recent call last):
  File "daily_file.py", line 245, in multiproc_buffer
    processing_line(input_dir, output_dir, year, month, day)
  File "daily_file.py", line 128, in processing_line
    mymoment = get_moment(radar, mykey, fillvalue=FILLVALUE)
  File "/home/548/vhl548/projects/RADAR_codes/cpol_level_2/scripts/processing_codes/utils.py", line 166, in get_moment
    mymoment = radar.fields[moment_name]
KeyError: 'cross_correlation_ratio'
Traceback (most recent call last):
  File "daily_file.py", line 245, in multiproc_buffer
    processing_line(input_dir, output_dir, year, month, day)
  File "daily_file.py", line 128, in processing_line
    mymoment = get_moment(radar, mykey, fillvalue=FILLVALUE)
  File "/home/548/vhl548/projects/RADAR_codes/cpol_level_2/scripts/processing_codes/utils.py", line 166, in get_moment
    mymoment = radar.fields[moment_name]
KeyError: 'cross_correlation_ratio'
Traceback (most recent call last):
  File "daily_file.py", line 245, in multiproc_buffer
    processing_line(input_dir, output_dir, year, month, day)
  File "daily_file.py", line 128, in processing_line
    mymoment = get_moment(radar, mykey, fillvalue=FILLVALUE)
  File "/home/548/vhl548/projects/RADAR_codes/cpol_level_2/scripts/processing_codes/utils.py", line 166, in get_moment
    mymoment = radar.fields[moment_name]
KeyError: 'cross_correlation_ratio'
